### PR TITLE
Update to python 3.6

### PR DESCRIPTION
**Pull request by denmat**
Thanks for your work on adding SQLAlchemy. This was really helpful.

Here are some updates we made to your fork. Really useful for us, maybe not so for you.

* Adds docker for easy testing
* Adds a `retry` to check the db exists before creating tables
* Allows headless testing to succeed
* Updates to our latest python

### CODE REVIEW

File: dejavu/database.py [hunk 1]
```diff
@@ -1,8 +1,10 @@
 from __future__ import absolute_import
 import binascii
+from tenacity import *
```
Comment (bcollazo) [inline dejavu/database.py]: Could we import explicitly instead? `from tenacity import retry`

File: setup.py [hunk 2]
```diff
@@ -15,7 +15,7 @@ def parse_requirements(requirements):
 
 
 PACKAGE_NAME = "PyDejavu"
-PACKAGE_VERSION = "0.1.3"
+PACKAGE_VERSION = "1.3.1"
```
Comment (bcollazo) [inline setup.py]: Do we need this update?

**Review by bcollazo — COMMENTED** [review]
Hey, this looks awesome; thanks for the help!

Question, why the CMD "tail -f /dev/null" stuff? Could you also add some small commands/instructions on how to use the docker stuff?  The README.md is probably a good place for it.

File: setup.py (hunk 2 above)
Comment (denmat) [reply setup.py]: We are calling this app in via another wrapper, we needed this to point to our latest release for it to install correctly.

**Comment by bcollazo** [comment]
LG! Thanks for this.

**Pull request merged**